src/components/pool_flux_chart.vue

```diff
@@ -511,23 +565,25 @@ export default {
   $darkGrey: #6E6E6E; //5:1 contrast
   #page-content {
     display: block;
+    max-width: 1500px;
+    margin: 0 auto;
   }
   #chart-container {
-    height: 70vh;
-    // width: 90vw;
+    min-height: 400px;
+    height: 75vh;
+    width: 90vw;
+    max-width: 1500px;
     margin-top: 1vh;
-    margin-bottom: 2vh;
+    margin-bottom: 1vh;
   }
   #caption-container {
     display: block;
   }
   .poolText {
     color: $poolColor;
-    font-weight: 500;
   }
   .fluxText {
     color: $fluxColor;
-    font-weight: 500;
```

Comment:
I think it could work well to bump the weight a little bit and make "pools" and "fluxes" pop throughout